src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/generate.ps1

```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider  "cognitiveservices/data-plane/LUIS/Runtime" -AutoRestVersion "latest"
\ No newline at end of file
+Start-AutoRestCodeGeneration -ResourceProvider  "cognitiveservices/data-plane/LUIS/Runtime" -AutoRestVersion "latest" -ConfigFileTag "runtime_3_0_preview"
```

Comment:
Why is this tag set as default in the readme.md

Reply:
What readme.md? Do you mean the generate.ps1? If yes, then because in the specs repo I kept the stable version as the default and added this here to show which tag was used in the generation.

Reply:
Referring to [this](https://github.com/moabba/azure-rest-api-specs/blob/708409d928fe5997dda0acb005cd1d0fdff1c47b/specification/cognitiveservices/data-plane/LUIS/Runtime/readme.md) readme.md file.

Reply:
Nope, no reason. The reviewer on other PR didn't raise up this, so it wasn't changed.

Reply:
@moabba Please undo these changes and regenerate the code